content/en/docs/concepts/instrumentation/_index.md

```diff
@@ -12,32 +12,41 @@ from the system's components must emit [traces](/docs/concepts/signals/traces/),
-used an APM agent to extract telemetry from your application, Automatic
-Instrumentation will give you a similar out of the box experience.
-
-To facilitate the instrumentation of applications even more, you can
-[manually instrument](manual/) your applications by coding against the
-OpenTelemetry APIs.
-
-For that you don't need to instrument all the dependencies used in your
-application:
-
-- some of your libraries will be observable out of the box by calling the
-  OpenTelemetry API themselves directly. Those libraries are sometimes called
-  **natively instrumented**.
-- for libraries without such an integration the OpenTelemetry projects provide
-  language specific [Instrumentation Libraries][]
-
-Note, that for most languages it is possible to use both manual and automatic
-instrumentation at the same time: Automatic Instrumentation will allow you to
-gain insights into your application quickly and manual instrumentation will
-enable you to embed granular observability into your code.
-
-The exact installation mechanism for [manual](manual/) and
-[automatic](automatic/) instrumentation varies based on the language you’re
-developing in, but there are some similarities covered in the sections below.
-
-[instrumentation libraries]:
-  /docs/specs/otel/overview/#instrumentation-libraries
+OpenTelemetry provides [APIs and SDKs for eleven languages](/docs/languages) to
+facilitate the process of instrumenting your code. Among others it provides the
+following benefits:
+
+- Each language-specific implementation of the API and SDK follows the
+  requirements and expectations of the
+  [OpenTelemetry specification](/docs/specs/otel/)
+- Libraries can leverage the OpenTelemetry API as a dependency, which will have
+  no impact on applications using that library, unless the OpenTelemetry SDK is
+  imported.
+- For each [signal](/docs/concepts/signals) (traces, metrics, logs) you will
+  find methods to create them, process them and export them.
+- With [context propagation](/docs/concepts/context-propagation) being built
+  into the implementations, you can correlate signals, regardless of where they
+  are generated
+- [Resources](/docs/concepts/resources) and
+  [Instrumentation Scopes](/docs/concepts/instrumentation-scope) allow grouping
+  of signals, by different entities.
+- [Semantic Conventions](/docs/concepts/semantic-conventions) provide a common
+  naming schema that can be used for standardization across code bases and
+  platforms.
+
+As a [developer](/docs/getting-started/dev/) you are able to code against the
+OpenTelemetry API, to enrich your application (or library) with granular
+observability. For that, you only need to instrument your code, as your
+dependencies will either integrate OpenTelemetry themselves directly, or you can
+leverage
+[Instrumentation Libraries](/docs/specs/otel/overview/#instrumentation-libraries)
+for those without such an integration.
```

Comment:
```suggestion
As a [developer](/docs/getting-started/dev/) you can code against the
OpenTelemetry API to enrich your application or library manually for
more granular observability. For that, you only need to instrument your
code, as your dependencies can either integrate OpenTelemetry directly,
or you can leverage
[Instrumentation Libraries](/docs/specs/otel/overview/#instrumentation-libraries)
for those without such an integration.
```

Could we drop a "manually" here somewhere, to remember the old days? ;) I think in the context of an automatic or zero-code instrumentation doc, this makes sense.

Reply:
I personally have a bigger issue with "manually" vs "automatic". "Manual" always comes with that connotation that it requires a lot of work and effort. But I am not strictly against it.